mllib/src/test/scala/org/apache/spark/ml/classification/LinearSVCSuite.scala

```diff
@@ -223,6 +229,25 @@ class LinearSVCSuite extends SparkFunSuite with MLlibTestSparkContext with Defau
     assert(model1.coefficients ~== coefficientsSK relTol 4E-3)
   }
 
+  test("linearSVC L-BFGS comparison with R e1071 and scikit-learn") {
+    val trainer1 = new LinearSVC().setSolver(LinearSVC.LBFGS)
+      .setRegParam(0.00003)
```

Comment:
Indeed. I can use your help here since I cannot find the theory proof for this case.